lib/matplotlib/widgets.py

```diff
@@ -152,7 +152,7 @@ class Button(AxesWidget):
     """
 
     def __init__(self, ax, label, image=None,
-                 color='0.85', hovercolor='0.95'):
+                 color='0.85', hovercolor='0.95', useblit=False):
```

Comment:
make it kwonly?

Reply:
Yes, we should maybe also do it for the other arguments, but that can be a different PR.